packages/ui-components/src/components/common/animation/Instructional.vue

```diff
@@ -0,0 +1,140 @@
+<template>
+  <div
+    class="relative aspect-square w-full h-full max-w-[250px] mx-auto mb-8 border-t border-r border-outline-3 select-none"
+  >
+    <div
+      class="absolute z-50 text-foreground dark:text-foundation"
+      :style="{
+        transitionProperty: 'all',
+        top: mousePosition.top + '%',
+        left: mousePosition.left + '%',
+        transitionDuration: animationDuration + 'ms'
+      }"
+    >
+      <ClickIcon
+        class="absolute -top-5 -left-4 h-12 w-12 -rotate-12 text-foreground"
+        :class="[{ hidden: !isClicked }]"
+      />
+      <MouseIcon class="absolute inset-0 h-11 w-11" />
+    </div>
+    <div class="w-full h-full overflow-hidden">
+      <slot name="background"></slot>
+      <template v-if="slotVisibility.slot1">
```

Comment:
5 slots seems like an arbitrary amount, remember that slots can be dynamically defined (like you once did for the Table component) and we can use that to our advantage here. Or maybe have 1 slot with the slot index being passed through slot props. And the dev can ask for how many slots he likes through a prop, or something